pkgs/applications/misc/slides/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "slides";
+  version = "0.1.1";
```

Comment:
[0.1.2](https://github.com/maaslalani/slides/releases/tag/v0.1.2) seems to be the latest release.